src/yachtzee.ts

```diff
@@ -675,11 +675,14 @@ export function yachtzeeChainDiet(simOnly?: boolean): boolean {
     (3 - get("_sweatOutSomeBoozeUsed", 0));
   const spleenAvailable = currentSpleenLeft + filters;
   const organsAvailable = fullnessAvailable + inebrietyAvailable + spleenAvailable;
-  const sufficientOrgansFor30 =
-    organsAvailable - synthCastsToCoverRun - extroSpleenSpace >= 30 + 5 * toInt(havePYECCharge);
-  const sufficientOrgansFor20 =
-    organsAvailable - synthCastsToCoverRun - extroSpleenSpace >= 20 + 5 * toInt(havePYECCharge);
-  const baseYachtzeeTurns = sufficientOrgansFor30 ? 30 : sufficientOrgansFor20 ? 20 : 10;
+
+  const cleanableSpleen = organsAvailable - synthCastsToCoverRun - extroSpleenSpace;
+  const sufficientOrgansFor = (yachtzees: number) =>
+    cleanableSpleen >= yachtzees + (havePYECCharge ? 5 : 0);
+  const possibleBaseYachtzeeTurns = [30, 20, 10];
+  const baseYachtzeeTurns = possibleBaseYachtzeeTurns.reduce((a, b) =>
+    sufficientOrgansFor(a) ? a : b
+  );
```

Comment:
This feels more like a `find` than a `reduce` situation;

```
const baseYachtzeeTurns = [30, 20, 10].find(sufficientOrgansFor) ?? 0;
```
Or something like that. This is assuming that 0 is the default. You could also do 
```
const baseYachtzeeTurns = [30, 20].find(sufficientOrgansFor) ?? 10;
```

It won't let me make this as a formal suggestion

If you do this, leave an empty line after the `sufficientOrgansFor` function, I think it'll be cleaner looking.